src/main/java/crusader/TaskList.java

```diff
@@ -0,0 +1,134 @@
+package crusader;
+
+import crusader.exception.CrusaderException;
+import crusader.exception.CrusaderNoSuchTaskException;
+
+import crusader.task.Task;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+/**
+ * A set of tasks to do
+ */
+public class TaskList {
+    /**
+     * The list of task
+     */
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Creates an empty tasklist
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Initializes a tasklist with a pre-existing list of tasks
+     * @param tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Describes the list of tasks in the task list.
+     */
+    @Override
+    public String toString() {
+        String returnString = "";
+        for (int x = 0; x < tasks.size(); x++) {
+            returnString = String.format("%s%d. %s\n", returnString, x + 1, tasks.get(x).toString());
```

Comment:
Love the way you adhere to the line limit and push extra characters to the next line